service/src/main/java/uk/gov/hmcts/reform/fpl/service/HearingBookingService.java

```diff
@@ -26,12 +26,12 @@
         }
     }
 
-    public HearingBooking getMostUrgentHearingBooking(CaseData caseData) {
-        if (caseData.getHearingDetails() == null) {
+    public HearingBooking getMostUrgentHearingBooking(List<Element<HearingBooking>> hearingBookings) {
+        if (hearingBookings == null) {
             throw new IllegalStateException("Hearing booking was not present");
```

Comment:
should we really be throwing an exception? Will this block doc generation etc?

Reply:
Agreed to look into this as tech debt.